<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>security</class>
 <widget class="QWidget" name="security">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>2048</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(251, 248, 239);</string>
  </property>
  <widget class="QPushButton" name="pBtnOK">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>下 一 步</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbx1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>800</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(219, 210, 200);
border-radius: 5px; </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lblPswd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>密  码</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldtPswd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(219, 210, 200);
border: 1px solid gray;
border-radius: 5px; </string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>enter password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblSubTitle1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>第 一 步</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>验证码</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldtCode">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(219, 210, 200);
border: 1px solid gray;
border-radius: 5px; </string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>enter verification code</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblFormula">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>验证码</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnChange">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(219, 210, 200);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(203, 191, 180);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
    </property>
    <property name="text">
     <string>换一道</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblimg1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/steps/images/step1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblimg2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/steps/images/step2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbx2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>800</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(219, 210, 200);
border-radius: 5px; </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lblSubTitle2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>第 二 步</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblAnswer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>密保答案</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldtAnswer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(219, 210, 200);
border: 1px solid gray;
border-radius: 5px; </string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>enter security answer</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblQuestion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>密保问题</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldtQuestion">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(219, 210, 200);
border: 1px solid gray;
border-radius: 5px; </string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>enter security question</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblSecurity">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>85</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>修改密保</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRet">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>返回上一级</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/interface/images/home.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>95</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/icon.jpeg);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pBtnOK</zorder>
  <zorder>lblimg1</zorder>
  <zorder>lblimg2</zorder>
  <zorder>lblSecurity</zorder>
  <zorder>pBtnRet</zorder>
  <zorder>lblTitle</zorder>
  <zorder>lblIcon</zorder>
  <zorder>gbx1</zorder>
  <zorder>gbx2</zorder>
 </widget>
 <tabstops>
  <tabstop>ldtPswd</tabstop>
  <tabstop>ldtCode</tabstop>
  <tabstop>ldtQuestion</tabstop>
  <tabstop>ldtAnswer</tabstop>
  <tabstop>pBtnOK</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ldtPswd</sender>
   <signal>returnPressed()</signal>
   <receiver>ldtCode</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ldtQuestion</sender>
   <signal>returnPressed()</signal>
   <receiver>ldtAnswer</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
